core/src/main/java/org/stellar/anchor/auth/Sep6MoreInfoUrlJwt.java

```diff
@@ -0,0 +1,23 @@
+package org.stellar.anchor.auth;
+
+import static org.stellar.anchor.auth.JwtService.CLIENT_DOMAIN;
+import static org.stellar.anchor.auth.JwtService.CLIENT_NAME;
+
+import io.jsonwebtoken.Jwt;
+import org.stellar.anchor.api.exception.SepException;
+
+public class Sep6MoreInfoUrlJwt extends AbstractJwt {
```

Comment:
nit: There's nothing specific to SEP-6 about `more_info_url` JWT, can we merge this with `Sep24MoreInfoJwt` and call it `MoreInfoJwt`?

Reply:
We need to differentiate the JWT (sep6/sep24) so that we know which secret to use when encoding